modules/tester.js

```diff
@@ -1136,6 +1137,11 @@ Tester.prototype.done = function done() {
     /*jshint maxstatements:20, maxcomplexity:20*/
     var planned, config = this.currentSuite && this.currentSuite.config || {};
 
+    this.casper.echo(f("done called"), "INFO");
```

Comment:
I am not sure if we really want to add the INFO echo messages contained in this PR. @n1k0, @hexid, @paazmaya ?